cli/api.js

```diff
@@ -62,14 +63,19 @@ function save() {
 }
 
 function existsOnNpm(plugin) {
-  plugin = plugin.split('#')[0].split('@')[0];
-  return npmName(plugin).then(unavailable => {
-    if (unavailable) {
+  const name = plugin.split('#')[0].split('@')[0];
+  return got
+    .get(registryUrl + name.toLowerCase(), {timeout: 10000, json: true})
+    .then(res => {
+      if (!res.body.versions) {
+        throw new Error();
```

Comment:
Error should be defined with a message.

Reply:
Meh, just used to get into the `.catch` block here. Could alternatively `return Promise.reject();`.

Reply:
Your ready to go. 🚀